airflow/www/templates/airflow/dags.html

```diff
@@ -403,7 +405,7 @@ <h2>DAGs</h2>
           container: "body",
         });
       });
-      d3.json("{{ url_for('Airflow.task_stats') }}", function(error, json) {
+      d3.json("{{ url_for('Airflow.task_stats') }}?dag_ids=" + all_dags_ids.join(','), function(error, json) {
```

Comment:
is `,` allowed in DAGids? 😁 

Also this parameter needs to be URI escaped, or bettwer yet if `d3.json` supports it `d3.json("{{ url_for('Airflow.task_stats') }}", {"dag_ids": all_dags_ids.join(',') }, function(error, json) {`

Reply:
Passing get parameters like you suggested above wasn't supported by d3.json so instead I've encoded each dag_id. I thought about passing the`dag_id=` multiple times but it does result in a rather long GET header (which I believe is often capped at 8kb?). This could potentially overflow for users with 100 dags per page.

Reply:
I think, we can use `request.getlist("dag_ids")` and send all request in seperate parameter e.g. `url-view?dag_ids=AAA&dag_ids=AAA`. Parameters should be encoded and so independent.

Reply:
Hey sorry this got lost in my inbox. I still have to provide a PR for the /blocked endpoint. Happy to provide an additional PR switching the rest of the end points to use getlist() if you guys prefer it?